src/main/frontend/format/block.cljs

```diff
@@ -360,7 +360,7 @@
              {:uuid uuid
               :content content
               :anchor (str uuid)
-              :level 2
+              :level 0
```

Comment:
this def looks like a bug either way?

Reply:
Sorry for the late reply, it's not a bug, in the idea database where we don't need to care about the final output text, the level could be 1. This `pre-block` is the top content above the first heading, currently, we treat it as the same level as the next heading.
We set the initial level to 2 so that the next heading's level would be 2, the font for H1 will be too large when reading on GitHub. 

We're doing some refactoring here: https://github.com/logseq/logseq/pull/1191, basically it'll use a dummy block to represent a page.